doc/whats-new.md

```diff
@@ -1,385 +1,205 @@
+
+There was a major overhaul of the secondary perils module
+(see https://github.com/gem/oq-engine/pull/8982)
+including the following regional liquefaction models:
+
+ZhuEtAl2017LiquefactionCoastal and ZhuEtAl2017LiquefactionGeneral,
+RashidianBaise2020Liquefaction, AllstadtEtAl2022Liquefaction,
+AkhlagiEtAl2021LiquefactionA and AkhlagiEtAl2021LiquefactionB
+Bozzoni2021LiquefactionEurope, TodorovicSilva2022NonParametric.
+
+The TodorovicSilva2022NonParametric model is based on a machine 
+learning approach and therefore now the engine supports the onnx runtime.
+
+# Bug fixes and better error messages
+
+Parallelizing engine calculations with pytest, GNU parallel or other
+approaches did not work due to conflicts in the calculation IDs (i.e.
+two calculations could get the same ID). This issue has been fixed.
+
+The internal output `avg_gmf` was computed incorrectly due to an ordering
+bug in the event ID. This has been fixed.
+
+There was a regression in engine 3.17 breaking the disaggregation by source
+for the Japan model. It has been fixed.
+
+We fixed a bug in calculations with a filtered site collection using
+the HM2018CorrelationModel. The model should still be considered
+experimental.
+
+When configured for multiple users the WebUI allowed people to see
+the results produced by other users knowing the URL. This is not
+possible anymore.
+
+Removing a calculation through the WebUI failed on Windows due to the
+use of a wrong path separator. This has been fixed.
+
+The command `oq reaggregate` was failing with a cryptic error message
+when starting from a calculation with a missing or inconsistent
+`aggregate_by`. Now there is a clear error.
+
+The command `oq engine --delete-calculation XXX` could not remove the
+calculation file `calc_XXX.hdf5` when the DbServer was on a different
+machine, or when the DbServer user had no permissions to delete. Now
+the deletion is performed directly by the user running the command.
+
+We fixed consequence calculations for 'homeless', 'fatality' and 'injury'
+with and without a specified 'time_event'.
+
+There is now a better error message for logic trees with branchsets
+exceeding the limit of 183 branches.
+
+There was a `ValueError: max() arg is an empty sequence` caused by
+sources below the `minimum_magnitude`: now they are simply discarded.
+
+We raise now a clear error message "quantiles are not supported with
+collect_rlzs=true" when the calculation of quantiles is not supported.
+
+We fixed a log message that was displaying a wrong number for
+the size of the produced GMFs in event based calculations.
+
+A site model can be split in multiple files. There is now a check on
+the headers and an error is raised if the headers are not consistent.
+
+# AELO project
+
+There was a substantial amount of work on the AELO project. The engine
+is finally able to compute the ASCE-41 and ASCE-7 parameters by using
+code from the USGS (the `rtgmpy` library).
```

Comment:
let's keep an experimental (or similar) disclaimer on this part for now 
@mmpagani @ManuelaVillani

Reply:
Done